From 609b147513f7b15de9e6043c16d6e8cb1ce7efb8 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Mon, 17 Apr 2017 02:58:22 +0100
Subject: [PATCH 3/3] Disable disastrous NET_WM_PING deletion dialogs

These cause more hassle than they're worth, and break many SDL games.
These apps are just slower to start (Thunderbird) or GL-locked (SDL),
and throwing up a "not responsive" window to break focus, input, etc,
when it clearly *is* working, just isn't helpful to anyone.

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 src/core/delete.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/core/delete.c b/src/core/delete.c
index 56ffa5c..bd235a7 100644
--- a/src/core/delete.c
+++ b/src/core/delete.c
@@ -98,6 +98,8 @@ show_delete_dialog (MetaWindow *window,
               "Got delete ping timeout for %s\n",
               window->desc);
 
+  return;
+
   if (window->dialog_pid >= 0)
     {
       present_existing_delete_dialog (window, timestamp);
-- 
2.12.2

